lib/lois/ci/travis.rb

```diff
@@ -11,7 +11,7 @@ def repository
 
       def commit_sha
         sha = ENV.fetch('TRAVIS_PULL_REQUEST_SHA')
-        sha.length == 0 ? ENV.fetch('TRAVIS_COMMIT') : sha
+        sha.empty? ? ENV.fetch('TRAVIS_COMMIT') : sha
```

Comment:
I presume this is a rubocop fix?

Reply:
That is correct, rubocop recommended this fix.